build.sh

```diff
@@ -366,79 +366,43 @@ function build_android {
         archive_android "Release"
     fi
 
-    cd android/filament-android
```

Comment:
In this file, take a look at the `build_clean` function, since it might have paths that need to be adjusted.

Reply:
Already did, it's all good :)